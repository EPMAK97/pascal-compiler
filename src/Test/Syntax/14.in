if a < b
    then: a := -a;