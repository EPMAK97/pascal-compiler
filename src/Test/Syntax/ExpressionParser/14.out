Error in pos 1:3 expected identifier, constant or expression 
